fff

```diff
@@ -95,6 +98,8 @@ key() {
 
 main() {
     shopt -s nocaseglob nullglob checkwinsize
+    [[ "$OSTYPE" == haiku ]] &&
+        : "${FFF_TRASH:=$(finddir B_USER_CACHE_DIRECTORY)/fff/trash/}"
```

Comment:
There's no way of doing this without `finddir`?

Reply:
I did some googling and I guess this is needed. :+1:

Reply:
I just noticed that `$XDG_CONFIG_HOME` is automatically set in Haiku. Perhaps `$FFF_TRASH` could be replaced with it + `/trash`? Or at least fall back to it if `$FFF_TRASH` is not set and it is; seems like a good thing to honor even if not on Haiku, especially since you're using `xdg-utils` by default.

I'd make a PR, but I just noticed the addition of `$FFF_CD_FILE` and would also like it to be moved to one of the XDG directories and I'm not sure which one (`$XDG_DATA_HOME`?).

Reply:
#50